DirectProgramming/DPC++FPGA/ReferenceDesigns/cholesky_inversion/src/memory_transfers.hpp

```diff
@@ -0,0 +1,159 @@
+#ifndef __MEMORY_TRANSFERS_HPP__
```

Comment:
if it is identical, why don't we move it to be a sibling of `streaming_cholesky.hpp` and `streaming_cholesky_inversion.hpp`?

Reply:
The include directory is not meant to house every file that's common to multiple samples, the bar is a bit higher, and I'm not sure the memory_transfers kernel is sufficiently generic and reusable to be worth putting in there.  If we are going to elevate it to the shared include folder, it probably needs a bit of work.

Reply:
The function ```VectorReadFromPipeToDDR``` in this cholesky_inversion design is added to the ```memory_transfers.hpp``` file compared to the cholesky one.
Agreed it would need more polish and to cover all the variants (matrix from DDR to local/matrix from local to DDR/vector from DDR to local/vector from local to DDR)